rabbitmq/configure.sls

```diff
@@ -2,6 +2,10 @@
 {% set rabbitmq_config = salt.pillar.get('rabbitmq:configuration') %}
 {% set rabbitmq_config_dir = '/etc/rabbitmq' %}
 
+set_UTF-8_locale:
+  cmd.run:
```

Comment:
We should use `locale.system` instead of a `cmd.run` call. 
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.locale.html#module-salt.states.locale